### PR TITLE
LogIn, Footer and Header components initial design

### PR DESCRIPTION
**Pull request by chrisT-pixel**
Co-Authored-By: christ-pixel <christ-pixel@users.noreply.github.com>

### CODE REVIEW

**Comment by chrisT-pixel** [comment]
seeking approval to merge logIn-and-init into master

File: src/containers/App/App.js [hunk 1]
```diff
@@ -4,6 +4,11 @@ import Layout from '../Layout/Layout';
 import LogIn from '../LogIn/LogIn';
 import EnrolmentView from '../EnrolmentView/EnrolmentView';
 
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faChevronRight, faChevronLeft, faEnvelope } from '@fortawesome/free-solid-svg-icons';
+
+library.add(faChevronRight, faChevronLeft, faEnvelope)
```
Comment (FedorGabrus) [inline src/containers/App/App.js]: Do we need fontawesome import in App.js?

**Comment by chrisT-pixel** [comment]
I'll take care of this tomorrow mate

**Comment by chrisT-pixel** [comment]
made the updates (besides removing style sheet as program would not compile without it)

File: src/components/LogInInput/LogInInput.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import styles from './LoginInput.module.css';
+import styles from './Loginmodule.scss';
```
Comment (FedorGabrus) [inline src/components/LogInInput/LogInInput.js]: Please delete this import as well as the empty file it refers to

**Pull request merged**